### DISCUSSION TITLE
Advanced Graphics Summit: GPU Work Graphs: Welcome to the Future of GPU Programming

### DISCUSSION THREAD

**Discussion by giscus** [gdc-search/gdc-search.github.io#3] [General]
# Advanced Graphics Summit: GPU Work Graphs: Welcome to the Future of GPU Programming

https://gdc-search.github.io/

**Comment by gdc-search** [comment]
D3D12 Work Graph:
https://devblogs.microsoft.com/directx/d3d12-work-graphs/
